src/main/java/de/tum/cit/aet/artemis/exercise/web/ParticipationResource.java

```diff
@@ -421,7 +421,8 @@ else if (exercise instanceof ProgrammingExercise) {
         // Process feedback request
         StudentParticipation updatedParticipation;
         if (exercise instanceof TextExercise) {
-            updatedParticipation = textExerciseFeedbackService.handleNonGradedFeedbackRequest(participation, (TextExercise) exercise);
+            StudentParticipation finalParticipation = participation;
```

Comment:
Since the endpoint works with one specific exercise id we should throw an error. In a best case scenario as soon as we know someone requests feedback for a text exercise while the module is disabled.